products/opensearch.md

```diff
@@ -29,6 +29,8 @@ releases:
 
 > [OpenSearch](https://opensearch.org/) is a community-driven, Apache 2.0-licensed open source search and analytics suite that makes it easy to ingest, search, visualize, and analyze data. Developers build with OpenSearch for use cases such as application search, log analytics, data observability, data ingestion, and more.
 
+OpenSearch is a fork of [Elasticsearch](https://endoflife.date/elasticsearch) 7.10 and provides backwards compatibility with the REST APIs for ingest, search, and management. OpenSearch is backwards compatible with Elasticsearch indices from versions 6.0 up to 7.10.
```

Comment:
@jamesfed @captn3m0 isnt this part kinda repeats itself ? i mean couldnt we make this much more cleaner?

Reply:
reworded this slightly, but this needs to be split since the sentences talk about different things (API vs index compatibility)